### PR TITLE
Add infoblox devices (Issue 2708)

### PR DESCRIPTION
**Pull request by wiad**
MIBS available at http://www.oidview.com/mibs/7779/md-7779-1.html

First attempt at adding device, not sure everything is kosher.

### CODE REVIEW

**Comment by f0o** [comment]
Can you provide a device for the CI to test against?
I can give you an IP we would be using or even setup a VPN to your aite to comply with data security policies of your company.
Preferably this particular comversation should be continued via email f0o@librenms.org, GPG available on keyservers :)

**Comment by laf** [comment]
@wiad You've got a php syntax issue in includes/definitions.inc.php The merge conflict was in that file and it's not been fixed. Look for >>>>> and you will see.

File: includes/discovery/os/nios.inc.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+if (!$os || $os === 'linux') {
+    if (stristr($sysObjectId, ".1.3.6.1.4.1.7779.1")) {
+        $os = 'nios';
+    }
+}
+
```
Comment (laf) [inline includes/discovery/os/nios.inc.php]: Lots of blank lines here, can you remove those please.

Comment (f0o) [inline includes/discovery/os/nios.inc.php]: I'll be around his work soon and try out the PR on his gear. Will do some fixes from there if needed.
Looking forward to another device being added :+1:

**Comment by paulgear** [comment]
:+1: from me; new device type, so shouldn't affect anything serious.

**Pull request merged**